docs/deploying.md

```diff
@@ -387,7 +531,7 @@ procedure. The directory includes an example `compose` configuration.
 
 2. Get the `docker-compose` binary.
 
-		$ sudo wget https://github.com/docker/compose/releases/download/1.1.0/docker-compose-`uname  -s`-`uname -m` -O /usr/local/bin/docker-compose
+		$ sudo wget https://github.com/docker/compose/releases/download/1.2.0/docker-compose-`uname  -s`-`uname -m` -O /usr/local/bin/docker-compose
```

Comment:
These are intended for the 1.7 release which the candidate is tagged as 1.2 I believe.  I'll double check with the team.